constructor/post-install.sh

```diff
@@ -8,4 +8,10 @@ python -m pip install ibis-vega-transform
 jupyter labextension install ibis-vega-transform
 jupyter labextension install @pyviz/jupyterlab_pyviz
 
+
+pip install \
+    git+https://github.com/ibis-project/ibis@10d40305defc7337d217b595c4b6b74e66d153da \
+    git+https://github.com/holoviz/holoviews.git@refs/pull/4517/head \
+    git+https://github.com/holoviz/hvplot.git@refs/pull/507/head
```

Comment:
This is probably not a great way to do this. It would better to create conda packages for these and then have the solver find those versions in constructor itself

Reply:
Do you mean something like a conda package on some anaconda account (maybe `quansight`) ?

The point here is that it should override the packages defined in the dependencies for this specific case. 
I agree, this approach is not good, but we just found this issue, and we will need that for running the notebooks for the presentation tomorrow.

Any extra thoughts considering this situation? thanks for reviewing this PR! :)

Reply:
I mean that you could have recipes for these packages in this repo and then build them "locally" on CI before running constructor.  Constructor should then get these versions of the deps

Reply:
thanks @scopatz for the explanation. I applied the suggestions you made (let me know if I'm doing it the same way you thought) ... I am just worried because it is taking too much time, more than 1h 21m and it is still in progress ...

Reply:
This looks great! Thanks Ivan!